package.json

```diff
@@ -91,7 +91,7 @@
     "@ndla/core": "^5.0.2",
     "@ndla/error-reporter": "^2.0.4",
     "@ndla/hooks": "^2.1.9",
-    "@ndla/icons": "^8.0.32-alpha.0",
+    "@ndla/icons": "^8.0.33-alpha.0",
```

Comment:
Hvorfor bumpet du ikke alle NDLA-pakkene?

Reply:
Jeg var helt sikker på at jeg hadde gjort det, men hadde en konflikt med master, så mulig rebasen ødela for meg 😅